openshift/README.md

```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
 1. In dev_console, `bayesian-data-importer` service is down.
 
 **cause**: In this case you have some data messed up in your Dynamo DB Tables.
-**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerboros_*`. 
+**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerberos_*`. 
```

Comment:
Use Same naming convention everywhere. 
If you are using {`YOUR_KERBEROS_ID`}, lets stick with it.

Reply:
This was existing documentation, I just corrected spelling here.